src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -342,6 +342,14 @@ void Rasterizer::UpdateDynamicState(const GraphicsPipeline& pipeline) {
     }
 }
 
+static s16 CombinedScissorValueTL(s16 scr, s16 win, s16 gen, s16 win_offset) {
```

Comment:
These can be lambdas in UpdateViewportScissorState

Reply:
Done